packages/translations/locales/de/dashboard.json

```
@@ -546,7 +643,17 @@
     "productMarketplace": "Produktmarktplatz",
     "productSuppliers": "Produktlieferanten",
     "products": "Produkte",
+    "findCandidates": "Kandidaten finden",
+    "findReplacements": "Ersatzpersonal finden",
+    "internPool": "Praktikantenpool",
```

Biome (2.4.15)

[error] The key internPool was already declared.

(lint/suspicious/noDuplicateObjectKeys)